cmake/Platform/Linux/Packaging/postinst.in

```diff
@@ -34,8 +34,8 @@ X-GNOME-Autostart-enabled=true\n\
     if [ "" != "$SUDO_USER" ]
     then
         pushd @CPACK_PACKAGING_INSTALL_PREFIX@
+        python/get_python.sh
```

Comment:
I have this section updated in another PR (https://github.com/o3de/o3de/pull/18406) which removes bootstrapping of python on windows. Doing a `CHOWN` here was a bad idea that needed to be corrected, so you can revert this particular file.